src/server/scripts/Northrend/IcecrownCitadel/boss_sister_svalna.cpp

```diff
@@ -568,7 +568,8 @@ struct npc_crok_scourgebane : public EscortAI
 
             for (uint32 itr = 0; itr < 4; ++itr)
                 if (Creature* crusader = ObjectAccessor::GetCreature(*me, _instance->GetGuidData(DATA_CAPTAIN_ARNATH + itr)))
-                    crusader->AI()->DoAction(ACTION_START_GAUNTLET);
+                    if (crusader->IsAlive())
+                        crusader->AI()->DoAction(ACTION_START_GAUNTLET);
```

Comment:
if you check the lines above at https://github.com/TrinityCore/TrinityCore/pull/24514/files#diff-834d6b45f7457540ff0fb85af4e413d2R548-R550 , the IsAlive() check is done for this script in the called DoAction() handler, not in the caller. Would it make sense to do the same for Crusader adding a IsAlive() check there ?

Reply:
true